cmd/src/gateway_benchmark.go

```diff
@@ -0,0 +1,258 @@
+package main
+
+import (
+	"crypto/tls"
+	"flag"
+	"fmt"
+	"io"
+	"net/http"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/sourcegraph/src-cli/internal/cmderrors"
+)
+
+const (
+	defaultRequestCount = 100
+	blue                = "\033[34m"
+	green       = "\033[32m"
+	yellow      = "\033[33m"
+	red         = "\033[31m"
+	reset       = "\033[0m"
```

Comment:
I think we should be using the stuff in color.go here: https://sourcegraph.sourcegraph.com/github.com/sourcegraph/src-cli/-/blob/cmd/src/colors.go